README.md

```diff
@@ -22,20 +20,158 @@ Or install it yourself as:
 
 ## Usage
 
-TODO: Write usage instructions here
+Run the install generator to get the config initializer
+
+```
+$ rails g open_stax:rescue_from:install
+```
+
+Declare that you want to use the openstax exception rescuer in your controller, preferably your `ApplicationController`
+
+```ruby
+class ApplicationController < ActionController::Base
+
+  # ...
+
+  use_openstax_exception_rescue
+
+  # ...
+
+  # Override the before and after hooks if you want to
+  # (See 'Controller before/after hooks')
+  #
+  # private
+  # def before_openstax_exception_rescue(exception_proxy)
+  #   # noop
+  # end
+  #
+  # def after_openstax_exception_rescue(exception_proxy)
+  #   respond_to do |f|
+  #     f.xml { render text: "I respond strangely to the XML format!",
+  #                    status: exception_proxy.status }
+  #   end
+  # end
+end
+```
+
+## Configuration
+
+This configuration, which is placed in `./config/initializers/rescue_from.rb` by the install generator, shows the defaults:
+
+```ruby
+OpenStax::RescueFrom.configure do |config|
+  config.raise_exceptions = ENV['RAISE_EXCEPTIONS'] || Rails.application.config.consider_all_requests_local
+
+  config.app_name = ENV['APP_NAME'] || 'Tutor'
+  config.app_env = ENV['APP_ENV'] || 'DEV'
```

Comment:
Yes, I decided it's going to be configurable because coming up with the magic way gets into messy loading order dependency problems. You wanna run an openstax server product, then dangit you can set an APP_NAME for it ;-P